### PR TITLE
test: Add Aurora + RSS integ test

### PR DESCRIPTION
**Pull request by charles-marion**
*Issue #, if available:*

*Description of changes:*
* Migrate AWS batch to use Fargate to speed up starting the hosts
* Add aurora workspace lifecycle intest test (add RSS and query it)
* Fixed RSS processing, the document type was passed incorrectly ({"S": "text/html"} instead of "text/html")

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: integtests/conftest.py [hunk 1]
```diff
@@ -19,7 +19,7 @@ def client(config):
     cognito = CognitoClient(
         region=region, user_pool_id=user_pool_id, client_id=user_pool_client_id
     )
-    email = "integ-test-user@example.local"
+    email = "integ-test-user@example.local-" + worker_id
```
Comment (charles-marion) [inline integtests/conftest.py:22]: Can ran the test in parallel, each thread will have its own user.

File: lib/shared/layers/python-sdk/python/genai_core/documents.py [hunk 2]
```diff
@@ -188,7 +188,11 @@ def delete_document(workspace_id: str, document_id: str):
     if not document:
         raise genai_core.types.CommonError("Document not found")
 
-    if document["status"] != "processed" and document["status"] != "error":
+    if (
+        document["status"] != "processed"
+        and document["status"] != "error"
+        and document["status"] != "enabled"  # rss feed final status
```
Comment (charles-marion) [inline lib/shared/layers/python-sdk/python/genai_core/documents.py:194]: Delete of rss feed was always returning  "Document not ready for deletion"

File: lib/shared/layers/python-sdk/python/genai_core/documents.py [hunk 3]
```diff
@@ -741,27 +745,47 @@ def batch_crawl_websites():
...
-                        if "crawler_properties" in post
-                        else True
-                    ),
-                    "limit": (
-                        int(post["crawler_properties"]["M"]["limit"]["N"])
-                        if "crawler_properties" in post
-                        else 250
-                    ),
-                    "content_types": (
-                        post["crawler_properties"]["M"]["content_types"]["L"]
```
Comment (charles-marion) [inline lib/shared/layers/python-sdk/python/genai_core/documents.py:760]: This value was incorrect. It's a list of [{S: "value"}]. Reworked.

File: integtests/chatbot-api/aurora_workspace_test.py [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+            metadata = json.loads(str(session.get("history")[1].get("metadata")))
+            found = len(metadata.get("documents")) > 0
+            break
+    client.delete_session(session_id)
+    assert found == True
+
+
+def test_delete_document(client: AppSyncClient):
+    if pytest.skip_flag == True:
+        pytest.skip("Open search is not enabled.")
```
Comment (charles-marion) [inline integtests/chatbot-api/aurora_workspace_test.py:178]:
```suggestion
        pytest.skip("Aurora is not enabled.")
```

**Review by bigadsoleiman — APPROVED** [review]

**Pull request merged**